docs/meshstack.aws.reserved-instance-guide.md

```diff
@@ -0,0 +1,27 @@
+---
+id: meshstack.aws.reserved-instance-guide
+title: Reserved Instances & Savings Plans Guide
+---
+
+An important piece of information regarding Reserved Instances to know before following this document, quoting from the AWS documentation,
```

Comment:
Maybe adding headings can improve this page, how about 

## Benefit Priority
## Centralized Purchases of RI/SPs
## Individual Purchases of RI/SPs

Reply:
Done.